ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -398,7 +398,7 @@ tests:
     cluster_profile: equinix-ocp-metal
     env:
       HYPERSHIFT_NODE_COUNT: "2"
-      ODF_OPERATOR_SUB_CHANNEL: stable-4.13
+      ODF_OPERATOR_SUB_CHANNEL: stable-4.15
```

Comment:
I'm unsure (let me highlight unsure ...) if stable-4.15 exists for now. From what I see in openshift release, 4.15 is dev-preview for the time being.

Reply:
then what is the channel ? dev-preview-4.15 ?